mdpow/workflows/dihedrals.py

```diff
@@ -491,9 +536,44 @@ def dihedral_violins(df, width=0.9, solvents=SOLVENTS_DEFAULT):
     axV.yaxis.set_visible(False)
     axV.spines["left"].set_visible(False)
 
+    axIM = g.axes_dict['Structure']                     
+    axIM.axis('off')
+
+    g.set_titles(plot_title)
+
     return g
 
-def plot_violins(df, resname, figdir=None, molname=None, width=0.9, solvents=SOLVENTS_DEFAULT):
+def build_svg(mol, molname=None, name=None, ab_pairs=None,
```

Comment:
Add a short doc string.

Any function that does not start with underscore is part of the public API and should be documented.